### PR TITLE
[IMP] accounting: completely new version of localization/mexico

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

**Comment by jcs-odoo** [comment]
For the reviewer:

This content comes from a document given by @vbe-odoo 
part of this content is first merged in 13.0 with the PR #852 

I'll rebase and resolve the conflicts before merging, but this content can already be reviewed.

Cheers

------
cc @ren-odoo

**Review by Feyensv — COMMENTED** [review]
Little tech review ;)

Furthermore, since you are modifying the majority of the file, you broke the titles hierarchy, you should also fix them to remove the warnings in the logs ;)
```
mexico.rst:1105: WARNING: Title level inconsistent:

What is the DIOT and the importance of presenting it SAT
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
mexico.rst:1120: WARNING: Title level inconsistent:

When to file the DIOT and in what format ?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
mexico.rst:1132: WARNING: Title level inconsistent:

Where the DIOT is presented ?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
mexico.rst:1148: WARNING: Title level inconsistent:

One more fact to know: the Batch load ?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
mexico.rst:1169: WARNING: Title level inconsistent:

How Generate this report in Odoo ?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by vbe-odoo** [comment]
Hi @jco-odoo !
Awesome work! Thanks for this!
I have only 1 doubt, and is about a section missing by the end called "Closing Fiscal Period"
This section was added on the V13 PR: https://github.com/odoo/documentation-user/blob/6559998ddd69e2cdd5eb95ffa007663b10614d89/accounting/fiscal_localizations/localizations/mexico.rst#closing-fiscal-period-in-odoo

But I don't know if this is going to be fp to this V14 documentation or if it is missing here.

Thanks!
Bests!
cc' @ren-odoo

**Comment by jcs-odoo** [comment]
> Little tech review ;)
> 
> Furthermore, since you are modifying the majority of the file, you broke the old [headings](https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_cheat_sheet.html#headings) specification. There are some warnings because of that in the logs. You should also fix those titles to remove the warnings :).
> 
> ```
> mexico.rst:1105: WARNING: Title level inconsistent:
> 
> What is the DIOT and the importance of presenting it SAT
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> mexico.rst:1120: WARNING: Title level inconsistent:
> 
> When to file the DIOT and in what format ?
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> mexico.rst:1132: WARNING: Title level inconsistent:
> 
> Where the DIOT is presented ?
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> mexico.rst:1148: WARNING: Title level inconsistent:
> 
> One more fact to know: the Batch load ?
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> mexico.rst:1169: WARNING: Title level inconsistent:
> 
> How Generate this report in Odoo ?
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> ```

Hello @Feyensv 
Thanks for the review.

I know these titles are broken, but I changed the title structure in the 13.0 PR (#852 ). So once that PR is merged and forward-ported to 14.0, the title structure will match.

I know there will be some conflicts to fix when I'll rebase. I probably should have left this PR in draft until the 13.0 is forward-ported.

Thanks :)

File: accounting/fiscal_localizations/localizations/mexico.rst [hunk 1]
```diff
@@ -2,323 +2,1027 @@
...
+   :alt: PDF external Trade
+
+Assign Pedimentos
+~~~~~~~~~~~~~~~~~
+
+If your company imports products and you need to add the **Pedimentos** number in your invoices, you
+can also configure Odoo to record the process.
+
+First, go to :menuselection:`Apps`, remove the "Apps" filter and search for `Mexico`, `mx` or
+`l10n_mx`. Then, install **Odoo Mexico Localization for Stock / Landing module
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply accounting/fiscal_localizations/localizations/mexico.rst:435]: done

File: accounting/fiscal_localizations/localizations/mexico.rst [hunk 2]
```diff
@@ -2,323 +2,1027 @@
...
+
+Taxes Configuration
+-------------------
+
+A necessary configuration for electronic invoicing to work correctly in Odoo is to add the factor
+type associated with sales taxes.
 
-Electronic Invoices (CDFI 3.2 and 3.3 format)
----------------------------------------------
+To make this configuration you first have to go to `Accounting --> Configuration --> Settings -->
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply accounting/fiscal_localizations/localizations/mexico.rst:169]: oopsie, thanks ;)
done

**Comment by jcs-odoo** [comment]
Conflicts resolved, and successfully rebased it with the content from #852 
Thanks @Feyensv for the fw-port :)

**Comment by jcs-odoo** [comment]
> Hi @jcs-odoo !
> Awesome work! Thanks for this!
> I have only 1 doubt, and is about a section missing by the end called "Closing Fiscal Period"
> This section was added on the V13 PR: https://github.com/odoo/documentation-user/blob/6559998ddd69e2cdd5eb95ffa007663b10614d89/accounting/fiscal_localizations/localizations/mexico.rst#closing-fiscal-period-in-odoo
> 
> But I don't know if this is going to be fp to this V14 documentation or if it is missing here.
> 
> Thanks!
> Bests!
> cc' @ren-odoo

Thanks @vbe-odoo 

It's normal. It's there, now that the branch is rebased with the changes done on 13.0.
I explained more to you by PM.

So, now, this PR just needs to be approved. Is someone reviewing it, content-wise? Maybe @ren-odoo ? 
The technical review is already done by @Feyensv .

Cheers,

Jonathan

**Comment by vbe-odoo** [comment]
Hi @jcs-odoo !
Thanks for the clarification. I can see the changes now after the merge in V13 and V14!
LGTM! I did a review and from my side, it looks good.
Can you give an extra pair of eyes on this @ren-odoo ? And then if it is all okay we can approve it from our side.
:)

Thanks!
Bests!

**Comment by ren-odoo** [comment]
Hi @vbe-odoo @jcs-odoo 
Thank you for the hard work, I did a general review on the content and it looks good to me. 
Well done!

**Review by ren-odoo — APPROVED** [review]

**Review by vbe-odoo — APPROVED** [review]
LGTM and check done also by @ren-odoo I will approve this PR.

**Pull request merged**